march_rqt_gait_selection/src/march_rqt_gait_selection/gait_selection_view.py

```diff
@@ -88,6 +88,18 @@ def add_subgait_menus(self, amount_of_new_subgait_menus):
 
     def update_version_menus(self):
         """When a gait is selected set the subgait labels and populate the subgait menus with the available versions."""
+        def version_sorter(version):
+            """Used in the sort function to sort numbers which are 9<.
+
+            :param version: str of the version
+            """
+            try:
+                length = len(version)
+                version_number = ''.join(letter for letter in version[length - 2: length] if letter.isdigit())
```

Comment:
Can you change letter to character, because the statement letter.isdigit() is a bit confusing and seems to be always false .

Reply:
this is indeed nicer